features/page_objects/base_page.rb

```diff
@@ -18,10 +18,10 @@ def submit(_args = {})
   end
 
   def accept_cookies
-    return unless has_accept_analytics_cookies?
+    return unless has_text? "Accept analytics cookies"
```

Comment:
Changing the element selectors should fix this issue without having to use capybara methods directly.

element(:accept_analytics_cookies, "[action='/cookies/accept_analytics'] [type='submit']")
element(:reject_analytics_cookies, "[action='/cookies/reject_analytics'] [type='submit']")
element(:hide_cookie_banner, "[action='/cookies/hide_this_message'] [type='submit']")